### ISSUE TITLE
How to start this project?

### ISSUE THREAD

**Issue by cnscorpions** [DanielRosenwasser/typescript-vue-todomvc#3]
I have tried npm run build, vue serve, but never works.
Seems to config uncorrectly

**Comment by thoni56** [comment]
Agreed, as a noob in this field I'd like at least a `scripts` section in the `package.json`. Please?

**Comment by jantaylor** [comment]
@cnscorpions @thoni56 in the project directory, use the cmd `npx webpack` then open `index.html`.
https://github.com/Microsoft/TypeScript-React-Conversion-Guide#typescript-react-conversion-guide

Or you can add:
```
// package.json
"scripts": {
  "build": "webpack"
},
```
Then call `npm run build` and then open the built `index.html` or `yarn build` if you use yarn.

**Comment by phoenixor** [comment]
i git clone the project to local, and run "npm i" to install the dependencies, but got wrong msg "can`t find vue module"
